bigbluebutton-html5/client/compatibility/kurento-extension.js

```diff
@@ -367,15 +362,23 @@ Kurento.prototype.startScreensharing = function () {
       return kurentoManager.exitScreenShare();
     };
   });
+  this.webRtcPeer.peerConnection.oniceconnectionstatechange = () => {
+    if (this.webRtcPeer) {
+      const connectionState = this.webRtcPeer.peerConnection.iceConnectionState;
+      if (connectionState === 'failed') {
```

Comment:
Should add a `closed` listener for both presenter and viewer peers, same thing as with webcams.